packages/rum-core/src/rumEvent.types.ts

```diff
@@ -678,7 +678,7 @@ export interface CommonProperties {
     /**
      * Type of the session
      */
-    readonly type: 'user' | 'synthetics'
+    readonly type: 'user' | 'synthetics' | 'ci_test'
```

Comment:
Did you run `yarn rum-events-format:sync` (cf file header)?
the `rum-events-format` ref should also be updated

Reply:
good catch